jpa/src/main/java/org/jboss/as/jpa/container/AbstractEntityManager.java

```diff
@@ -62,8 +62,6 @@
     public static final String PESSIMISTIC_WRITE_LOCK_MODE = "pessimistic_write";
     public static final String NONE_LOCK_MODE = "none";
 
-    private final transient boolean isTraceEnabled = ROOT_LOGGER.isTraceEnabled();
```

Comment:
Thanks, Scott. IMO the change in this particular commit is out of scope for this JIRA.